docs/website/references/cli/defradb_start.md

```diff
@@ -14,7 +14,12 @@ defradb start [flags]
 
 ```
       --allowed-origins stringArray   List of origins to allow for CORS requests
-      --development                   Enables a set of features that make development easier but should not be enabled in production
+      --development                   Enables a set of features that make development easier but should not be enabled 
```

Comment:
Suggestion: I think this reads a bit odd now "should not be enabled" and under production some bullets. Perhaps leave as was and still point out the differences that happen with flag is enabled

Reply:
On re-read, I think its just the odd line break before it says "in production:" on a new line?

Reply:
Ah i see.

I think something ugly but this should fix it?:

```go
const developmentDescription = `Enables a set of features that make development easier but should not be enabled` +
	` in production:` + `

- allows purging of all persisted data 
- generates temporary node identity if keyring is disabled
`

````